web-local/src/lib/util/file-upload.ts

```diff
@@ -234,3 +188,20 @@ async function getResponseFromModal(
     });
   });
 }
+
+export function openFileUploadDialog(multiple = true) {
+  return new Promise<Array<File>>((resolve) => {
+    const input = document.createElement("input");
+    input.multiple = true;
+    input.type = "file";
```

Comment:
What about using `input.accept = ".csv,.tsv,.parquet"`? Then I don't think we'd have to check for valid/invalid files above.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept

Reply:
Good idea. We still need the code for invalid files because it can be used with file drop.